static/js/dismiss_banner.js

```diff
@@ -0,0 +1,41 @@
+$(document).ready(function() {
+  function setCookie(name, value, days) {
+    let expires = "";
+    let date = new Date(); // Create a new Date object
+    let dateToSecond = 24 * 60 * 60 * 1000;
+

```

Comment:
nit: I would default `days` to some value, eg `1 + 4 * 365`, so that the cookie eventually expires. Otherwise we are relying on heuristics.